CHANGELOG.md

```diff
@@ -1,170 +1,212 @@
-- The light-blue, dark-blue, light-green, dark-green, light-purple, dark-purple, light-red, dark-red themes have been deprecated since 4.19. Please use light or dark instead, or create your own theme.
+•	decorators.declared deprecated since version 4.16. declared() is not needed to extend Accessor anymore. See Implementing Accessor for updated information.
+
+•	promiseUtils.reject deprecated since version 4.19. Use the native Promise.reject() method instead.
+
+•	promiseUtils.resolve deprecated since version 4.19. Use the native Promise.resolve() method instead.
+
+•	projection.isSupported deprecated since version 4.18.
+
+•	AttachmentInfo deprecated since version 4.19. Use AttachmentInfo instead.
+
+•	ChartMediaInfoValueSeries.x deprecated since version 4.17. Use value instead.
+
+•	ChartMediaInfoValueSeries.y deprecated since version 4.17. Use tooltip instead.
+
+•	SizeVariable.expression deprecated since version 4.2. Use SizeVariable.valueExpression instead.
+
+•	PathSymbol3DLayer.size deprecated since version 4.12. Use PathSymbol3DLayer.width or PathSymbol3DLayer.height instead.
+
+•	symbolPreview deprecated since version 4.11. Use symbolUtils instead.
+
+•	symbolPreview.renderPreviewHTML deprecated since version 4.11. Use symbolUtils.renderPreviewHTML instead.
+
+•	ClosestFacilityTask deprecated since version 4.20. Use closestFacility instead.
+
+•	FindTask deprecated since version 4.20. Use find instead.
+
+•	GeometryService deprecated since version 4.20. Use geometryService instead.
+
+•	Geoprocessor deprecated since version 4.20. Use geoprocessor instead.
+
+•	IdentifyTask deprecated since version 4.20. Use identify instead.
+
+•	ImageIdentifyTask deprecated since version 4.20. Use imageService instead.
+
+•	ImageServiceIdentifyTask deprecated since version 4.18. Use imageService instead.
+
+•	Locator deprecated since version 4.20. Use locator instead.
+
+•	PrintTask deprecated since version 4.20. Use print instead.
+
+•	QueryTask deprecated since version 4.20. Use query instead.
+
+•	RouteTask deprecated since version 4.20. Use route instead.
+
+•	ServiceAreaTask deprecated since version 4.20. Use serviceArea instead.
+
+•	AddressCandidate deprecated since version 4.20. Use AddressCandidate instead.
+
+•	AlgorithmicColorRamp deprecated since version 4.20. Use AlgorithmicColorRamp instead.
+
+•	AreasAndLengthsParameters deprecated since version 4.20. Use AreasAndLengthsParameters instead.
+
+•	AttachmentQuery deprecated since version 4.20. Use AttachmentQuery instead.
+
+•	BufferParameters deprecated since version 4.20. Use BufferParameters instead.
+
+•	ClosestFacilityParameters deprecated since version 4.20. Use ClosestFacilityParameters instead.
+
+•	ClosestFacilitySolveResult deprecated since version 4.20. Use ClosestFacilitySolveResult instead.
+
+•	ColorRamp deprecated since version 4.20. Use ColorRamp instead.
```

Comment:
the ones where the deprecation says to use the same name is weird.

Reply:
For the ones that look like duplicates, the path changed and the module name stayed the same.

`esri/tasks/support/ClosestFacilitySolveResult`

changes to:

`esri/rest/support/ClosestFacilitySolveResult`

And...

`esri/tasks/support/ColorRamp`

Changes to:

``esri/rest/support/ColorRamp``